tests/unit/CMakeLists.txt

```
@@ -0,0 +1,51 @@
+# Copyright (C) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+cmake_minimum_required(VERSION 3.14)
+
+project(wamr_unit_tests)
+
+include(CTest)
+
+if (NOT DEFINED WAMR_BUILD_INTERP)
+  # Enable Interpreter by default
+  set (WAMR_BUILD_INTERP 1)
```

```suggestion
  set(WAMR_BUILD_INTERP 1)
```
[nit] probably better to avoid the whitespaces in this file (not only this line) for consistency with other CMakeLists.txt in the repo